.travis.yml

```diff
@@ -48,16 +48,26 @@ matrix:
   allow_failures:
     - rust: nightly
   include:
-    - rust: stable
+    - name: Backend
+      rust: stable
       before_install:
-        - nvm install 10
-        - npm ci
         - rustup component add rustfmt-preview
         - rustup component add clippy-preview
       script:
         - cargo fmt -- --check
         - cargo clippy --all-targets --all-features --all
         - cargo test
+    - name: Frontend
+      rust: stable
+      before_install:
+        - nvm install 10
+        - npm ci
+      # skip diesel
+      install:
+        - echo
+      before_script:
+        - echo
```

Comment:
why the `echo` lines?

Reply:
To override the top-level install/before_script: but we probably can get rid of most of them. Let me update the PR.

Reply:
you can use `install: skip` in those cases (see https://docs.travis-ci.com/user/build-stages/#build-stages-and-deployments)

Reply:
ohh, didn't know that.